### PR TITLE
DOC: Limit the DIPY logo height in README for better rendering.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Limit the DIPY logo height in `README.rst` for a better rendering.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
@skoudoro @Garyfallidis IMHO the logo takes an excessive part of the homepage in GitHub. That's the motivation of this PR. The original file is 277px in height. I used 100px in this PR. I think reducing the size helps focusing on the text while preserving the identification of the toolkit with the logo.

If you are satisfied with the current rendering, please discard this PR.

Otherwise, although we may not be able to have a fair comparison, you may want to use this [on-line rST tool](http://rst.ninjs.org/?theme=basic#Li4gaW1hZ2U6OiBodHRwczovL2VsZWYuc29pYy5pbmRpYW5hLmVkdS9kaXB5L3N0YXRpYy9pbWFnZXMvZGlweS1sb2dvLnBuZwogIDpoZWlnaHQ6IDEwMHB4CiAgOnRhcmdldDogaHR0cDovL2RpcHkub3JnCiAgICAgOmFsdDogRElQWSAtIERpZmZ1c2lvbiBJbWFnaW5nIGluIFB5dGhvbgoKCgoKCg==) for comparison/preview purposes.

**Comment by Garyfallidis** [comment]
Yes, please change the height to 180. Also the link to http://dipy.org is not working.
Finally can you remove the header from the md file? Now it looks too small.
``
=======
DIPY
=======
``
Thank you @jhlegarreta !!

**Comment by jhlegarreta** [comment]
Done.

The **target issue** is one that Serge and I had identified: my guess was that it was may be forbidden for security reasons. Now I've checked that it was an alignment issue and **it's been fixed** !

Patch set 06187f2 works in an on-line rST tool:
http://bit.ly/2FNKu8l

File: README.rst [hunk 1]
```diff
@@ -1,10 +1,8 @@
 .. image:: doc/_static/dipy-logo.png
+  :height: 180px
   :target: http://dipy.org
-     :alt: DIPY - Diffusion Imaging in Python
+  :alt: DIPY - Diffusion Imaging in Python
 
-======
```
Comment (skoudoro) [inline README.rst]: The logo and badge are too close, Can you add a horizontal line to separate them? ` ___`.

Thanks!

Comment (jhlegarreta) [reply README.rst]: What about a vertical bar (`|`) to add some vertical space?  9f74e05 uses that. You can view the effect in the online rST editor.

Comment (skoudoro) [reply README.rst]: ok

**Pull request merged**